docs/installation.rst

```diff
@@ -54,33 +67,27 @@ More info can be found here: https://jax.readthedocs.io/en/latest/developer.html
 
 The following are instructions tested to work on the Traverse supercomputer at Princeton:
 
-.. code-block:: bash
+.. code-block:: sh
 
     git clone https://github.com/PlasmaControl/DESC.git
     cd DESC
-    module load anaconda3/2020.11 cudatoolkit/11.1 cudnn/cuda-11.1/8.0.4
 
+    module load anaconda3/2020.11 cudatoolkit/11.1 cudnn/cuda-11.1/8.0.4
 
-Then install python dependencies:
-
-.. code-block:: bash
-
-   conda create --name desc-env python=3.8 # create a new conda virtual environment
-   conda activate desc-env
-   # install what you can of the requirements with conda, ends up being all but jax, jaxlib and nvgpu
-    conda install colorama "h5py>=3.0.0" "matplotlib>=3.0.0,<=3.4.2" "mpmath>=1.0.0" "numpy>=1.2.0" "netcdf4>=1.5.4" psutil "scipy>=1.5.0" shapely termcolor
+    conda create --name desc-env python=3.8
```

Comment:
Don't we test on up to Python 3.9? If the IBM power architecture requires 3.8 for some reason, we should make an explicit note of that.

Reply:
Btw besides removing shapely and fixing a typo in the numpy version, I decided not to change the install instructions for the IBM power architecture as I can't test it.